build/yamls/base/conf/antrea-agent.conf

```diff
@@ -93,6 +93,10 @@ wireGuard:
 #  The port for WireGuard to receive traffic.
 #  port: 51820
 
+egress:
+#  The cidrs will ignore SNAT action when Egress enabled
```

Comment:
How about `exceptCIDRs is the CIDR ranges to which outbound Pod traffic will not be SNATed by Egress.`

Reply:
Agreed, fix it later.